test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -1700,13 +1699,10 @@ BOOST_AUTO_TEST_CASE(sha256)
 							 "  }\n"
 							 "}\n";
 	compileAndRun(sourceCode);
-	auto f = [&](u256 const& _input) -> u256
-	{
-		return dev::sha256(dev::ref(toBigEndian(_input)));
-	};
-	testSolidityAgainstCpp("a(bytes32)", f, u256(4));
-	testSolidityAgainstCpp("a(bytes32)", f, u256(5));
-	testSolidityAgainstCpp("a(bytes32)", f, u256(-1));
+	std::cerr << toHex(u256(4));
+	testSolidityAgainstCpp("a(bytes32)", string("e38990d0c7fc009880a9c07c23842e886c6bbdc964ce6bdd5817ad357335ee6f"), u256(4));
```

Comment:
This should be a hex string, right? So you probably need `fromHex("e38...")`. Did you run these locally?